### PR TITLE
Submission A0 and A1

### PR DESCRIPTION
**Pull request by cdhuyen**
This includes my solution for the Hello World program.

### CODE REVIEW

File: A0/cdhuyen-HelloWorld.py [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import argparse
```
Comment (s-will) [inline A0/cdhuyen-HelloWorld.py]: please add the she-bang line
```suggestion
#!/usr/bin/env python3
import argparse
```

File: A0/cdhuyen-HelloWorld.py [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+
+# Initialize the argument parser and parse the arguments
+parser = argparse.ArgumentParser()
+parser.add_argument("input", help="Input file")
+args = parser.parse_args()
+
+with open(args.input, "r") as file:
+    print("Hello World!")
+    for line in file:
+        print(line.strip())
```
Comment (s-will) [inline A0/cdhuyen-HelloWorld.py]: This would still print an additional newline, if the input file would not end with newline --- please rethink this

Comment (cdhuyen) [reply A0/cdhuyen-HelloWorld.py]: If there is a newline within in the text, e.g.
Hello

World
then should this newline also be removed?

Comment (s-will) [reply A0/cdhuyen-HelloWorld.py]: The idea is to print the file exactly as it is. [Hint: in Python, one can print without adding a newline]

Comment (cdhuyen) [reply A0/cdhuyen-HelloWorld.py]: I have updated the script. Maybe this is not elegant at all, but I tested it on a text file which does not end with a newline and it worked.

File: A0/cdhuyen-HelloWorld.py [hunk 3]
```diff
@@ -7,5 +8,8 @@
 
 with open(args.input, "r") as file:
     print("Hello World!")
-    for line in file:
```
Comment (s-will) [inline A0/cdhuyen-HelloWorld.py]: Sorry, I didn't intend this to get lengthy ;). The main trick is to use the end flag of print, like in
print(line, end='')
for line in file:
  print(line,end='')
should do the job. Even shorter
   print(file.read(),end='')
should work

Comment (cdhuyen) [reply A0/cdhuyen-HelloWorld.py]: Okay, I know the end flag of the print function, but the problem was that I thought we should remove all line breaks and empty lines after the END of the text content. That's why I did not use the end argument in the print function. But now I fully understand what the program should print.

**Comment by cdhuyen** [comment]
Ready for review :)

**Comment by cdhuyen** [comment]
Again, ready for review :)
I have added a main function (I don't know if it looks nice) and a short documentation section.

**Comment by s-will** [comment]
Nice that you took the time to improve the code -- looks very good :+1:

**Pull request merged**